src/lib/createStardustComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import createComponentInternal from './createComponent'
+import * as React from 'react'
+import { pick } from 'lodash'
```

Comment:
Always keep code consistent. It makes maintenance and contributions easier. import * as here. We'll optimize this in our move to a babel build.